app/controllers/pages_controller.rb

```diff
@@ -9,10 +9,8 @@ def home; end
   def find_placecal
     @grouped_sites = Site.where(is_published: true)
                          .joins(:primary_neighbourhood)
-                         .merge(Neighbourhood.order(district: :asc))
-                         .group_by{ |s| s.primary_neighbourhood.district }
-
-
+                         .merge(Neighbourhood.order(ancestry: :asc))
+                         .group_by { |s| s.primary_neighbourhood.parent.name }
```

Comment:
This block changes the ordering on neighbourhoods. We no longer have the `.district` attribute column so it's not really possible to refer to the parent district via SQL anymore. Instead I'm asking it to filter by Ancestry, which means the resulting ordering is no longer alphabetical